apps/src/music/views/Controls.jsx

```diff
@@ -59,56 +60,83 @@ const Controls = ({
     );
   };
 
-  const renderIconButton = (icon, onClick) => (
-    <div className={classNames(moduleStyles.controlButtons, moduleStyles.side)}>
-      <FontAwesome
-        icon={icon}
-        className={moduleStyles.iconButton}
-        onClick={onClick}
-      />
-    </div>
+  const renderIconButton = (icon, onClick, hide) => (
+    <button
+      className={classNames(
+        moduleStyles.controlButton,
+        moduleStyles.controlButtonIcon,
+        hide && moduleStyles.controlButtonHide
+      )}
+      onClick={onClick}
+      type="button"
+    >
+      <FontAwesome icon={icon} className={moduleStyles.icon} />
+    </button>
   );
 
-  const beatPadIconSection = renderIconButton('th', () => {
+  const beatPadIconButton = renderIconButton('th', () => {
     analyticsReporter.onButtonClicked('show-hide-beatpad', {
       showing: !isBeatPadShowing,
     });
     dispatch(toggleBeatPad());
   });
-  const infoIconSection = instructionsAvailable
-    ? renderIconButton('info-circle', toggleInstructions)
-    : null;
+
+  const infoIconButton = renderIconButton(
+    'info-circle',
+    toggleInstructions,
+    !instructionsAvailable
+  );
 
   const [leftIcon, rightIcon] = instructionsOnRight
-    ? [beatPadIconSection, infoIconSection]
-    : [infoIconSection, beatPadIconSection];
+    ? [beatPadIconButton, infoIconButton]
+    : [infoIconButton, beatPadIconButton];
 
   return (
     <div id="controls" className={moduleStyles.controlsContainer}>
       {isBeatPadShowing && renderBeatPad()}
-      {leftIcon}
       <div
-        className={classNames(moduleStyles.controlButtons, moduleStyles.center)}
+        className={classNames(moduleStyles.section, moduleStyles.sectionSide)}
+      >
+        {leftIcon}
+      </div>
+      <div
+        className={classNames(moduleStyles.section, moduleStyles.sectionCenter)}
       >
-        <FontAwesome
-          icon={isPlaying ? 'stop-circle' : 'play-circle'}
+        <button
+          className={classNames(
+            moduleStyles.controlButton,
+            moduleStyles.controlButtonRun
+          )}
           onClick={() => setPlaying(!isPlaying)}
-          className={moduleStyles.iconButton}
-        />
+          type="button"
+        >
+          <FontAwesome
+            icon={isPlaying ? 'stop' : 'play'}
+            className={moduleStyles.playStopIcon}
+          />
+          <div className={moduleStyles.text}>
+            {isPlaying ? commonI18n.stop() : commonI18n.runProgram()}
```

Comment:
If the goal is to align this button to other labs, should the label here be "reset" instead of "stop"?

Reply:
Yeah, wasn't sure about this. 'Reset' felt weird to me from a music playback perspective, but maybe it's not that confusing? Let me check with @moneppo